cobalt/devinfra/kokoro/bin/dind_build.sh

```diff
@@ -42,17 +42,33 @@ fi
 # Load common routines (we only need run_gn and ninja_build).
 . $(dirname "$0")/common.sh
 
-# Using repository root as work directory.
 WORKSPACE_COBALT="${KOKORO_ARTIFACTS_DIR}/git/src"
-cd "${WORKSPACE_COBALT}"
 
 configure_environment
 
-# Build Cobalt.
 pipeline () {
   local out_dir="${WORKSPACE_COBALT}/out/${TARGET_PLATFORM}_${CONFIG}"
-  run_gn "${out_dir}" "${TARGET_PLATFORM}" "${EXTRA_GN_ARGUMENTS:-}"
-  ninja_build "${out_dir}" "${TARGET}"
+  local src_dir="${KOKORO_ARTIFACTS_DIR}/chromium/src"
+  mkdir -p "${src_dir}"
+  cp -aT "${WORKSPACE_COBALT}" "${src_dir}"  # avoid CIPD invalid cross-device link
+
+  # gclient setup and sync
```

Comment:
nit: Line comments should be full sentences.

Reply:
I've never heard that and I'm inclined to disagree.
Is it mandated somewhere?

Reply:
The other comments in this file follow that format.

Reply:
go/cstyle#Punctuation,_Spelling_and_Grammar

Cobalt devs defer to C++ style guide wherever we can. It'd be helpful to indicate why we checkout depot_tools here (rather than configure Kokoro to pull it for us).